java/fory-core/src/main/java/org/apache/fory/builder/CodecBuilder.java

```diff
@@ -123,6 +123,8 @@ public CodecBuilder(CodegenContext ctx, TypeRef<?> beanType) {
   /** Returns an expression that serialize java bean of type {@link CodecBuilder#beanClass}. */
   public abstract Expression buildEncodeExpression();
 
+  public abstract Expression buildXlangEncodeExpression();
```

Comment:
Could we keep buildXlangEncodeExpression and buildXlangDecodeExpression in BaseObjectCodecBuilder.java?